### ISSUE TITLE
nan AIC on edited models

### ISSUE THREAD

**Issue by mrescalona** [dportik/dadi_pipeline#17]
Hi

I'm running 2 separate runs of dadi with data from 2 closely related populations: one containing all samples from both populations (31 and 24, respectiveley) and one containing a subset of 10 samples from each population with higher coverage (trying to mitigate the impact of missing data).

Both populations suffer from higher than average levels of inbreeding, with one of them having an absurd level of it. So I decided to include it in the models changing the fs in the models to fs = dadi.Spectrum.from_phi_inbreeding(phi, ns, (xx, xx), (F1, F2), (2, 2)), as describeb in dadi's manual. After making all changes into the scripts I've started running both of them.

For both dataset, unfortunalety, the summaries are returning AICs as nan. I thought it could be a result of the models being edited, but running with the default models return the same results.

Unlike the other issue placed here, ALL my AIC are nan and I don't get and error message. 

I assume it must be my data, but I don't know why.

Any advice would be appreciated.

Cheers,
Manuel Escalona

**Comment by dportik** [comment]
Hi Manuel,
That does seem like odd behavior. Perhaps a good sanity check would be to run a couple of the default models with the example datasets provided here. That can tell you if everything is working correctly (e.g., dadi version and all python packages are fine). If the default models run with the example dataset, then the problem is likely to be your empirical data. 

Let me know if the example datasets run okay, and we can troubleshoot from there.

**Comment by dportik** [comment]
@mrescalona do you have any update on the issue?

**Comment by mrescalona** [comment]
Really sorry for not aswering sooner.
I did had some problems with the example data. But since I was running the models on different computers, I noticed it wasn't on all the computers that the example data was also running into problems.

Long story short: I had a lot of conflicting libraries and dependencies on the computers, so I had to create a fresh environment and it resolved the issue for the example data.

For my dataset, however, the problems persisted. After more troubleshooting I found the error to be a mistake I did while editing the models and subsampling the data for the inbreeding function.

Sorry it took so long to reply.
